brainstormforce/ultimate-addons-for-gutenberg
# SPCTR-2795 Fix: Navigation slug option not displayed.

### Description
This change will fix the issue with the Navigation Slug option not appearing even when Hash Navigation is enabled. 

### Types of changes
Bug fix (non-breaking change which fixes an issue)

### How has this been tested?

1. Add a slider to the page/post. 
2. Turn ON Enable the Hash Navigation option from Settings -> General -> Slider Panel. 
3. When you go to individual slide settings, General Panel with Navigation Slug option is expected to appear. Earlier it was not appearing. This change has fixed that issue. 

### Checklist:
